<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicForm</class>
 <widget class="QWidget" name="MusicForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_music_picture">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_music_picture">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_music_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>我记得我爱过</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_music_singer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>何润东</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_like">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_share">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
